clients/cobol-lsp-vscode-extension/src/commands/CommentCommand.ts

```diff
@@ -63,10 +63,10 @@ export class ToggleComments {
             case CommentAction.COMMENT: return commentLine;
             case CommentAction.UNCOMMENT: return uncommentLine;
             case CommentAction.TOGGLE:
-                const anyNonComment = textLines
```

Comment:
Does it work also with debug indicators `d`?

Reply:
Yes, debug indicators are treated the same as continuation lines. I'd rather add this case into unit tests too.